test/publish.sh

```diff
@@ -65,6 +65,9 @@ dependencies:
   tools:
     git: https://github.com/istio/tools
     branch: master
+  envoy:
+    git: https://github.com/envoyproxy/envoy-wasm
```

Comment:
Shouldn't this be `istio/envoy`?

Reply:
This is because of the same reason you mentioned here: https://github.com/istio/release-builder/pull/335#issuecomment-664351623. Dry run will fail if this is set to istio/envoy. I think it is fine to use upstream envoy repo here, continuously syncing istio/envoy master just to make dry-run test pass seems unnecessary.

Reply:
ah ok, this is used only in testing, right? We should make sure to not try to push any tag to it :)